modules/ROOT/pages/platform/cloud-providers.adoc

```diff
@@ -96,6 +96,9 @@ When this is first set up, you should have no billing history.
 [discrete]
 ==== 4. Complete the set up
 
+pass:[<!-- vale Vale.Terms = NO -->]
+pass:[<!-- vale Neo4j.ParaNewLine = NO -->]
+
```

Comment:
I've added it myself to skip checks on the `MANAGE VIA NEO4J, INC.` line, which would otherwise raise warnings.
The next two lines (with a `YES` in place of `NO`) reactivate the checks.